roles/openshift_openstack/defaults/main.yml

```diff
@@ -16,8 +16,10 @@ openshift_openstack_nodes_to_remove: []
 openshift_openstack_cluster_node_labels:
   app:
     region: primary
+    pod_vif: nested-vlan
```

Comment:
Do we really want to add this by default to all deployments? Even the ones without Kuryr?

I don't think it would harm anything, but it may be a little confusing for admins who don't opt into this.

Reply:
Upss, sorry, you are right, it should not be added if kuryr is not enabled. I'll fix it!